nxdl.xsd

```diff
@@ -169,6 +172,32 @@
 		</xs:attribute>
 	</xs:attributeGroup>
 
+	<xs:attributeGroup name="nameTypeAttributeGroup">
+		<xs:attribute name="nameType" use="optional" default="specified">
+			<xs:annotation>
+				<xs:documentation>
+					This interprets the ``name`` attribute as:
+
+					* ``"specified"`` = Exactly this name.
+					* ``"any"`` = Any name not already used in group.
```

Comment:
Note that NeXus convention is not fully in line with the above statement default="specified".
To express that we may add here the following:  
Note that if no name and nameType are provided for a group, nameType="any" becomes the default.

Reply:
done per f9ceb0b